webapp/components/post_view/post_body_additional_content.jsx

```diff
@@ -187,12 +203,24 @@ export default class PostBodyAdditionalContent extends React.PureComponent {
                     onClick={this.toggleEmbedVisibility}
                 />
             );
-            const message = (
+
+            var message = (
                 <div key='message'>
                     {this.props.message}
                 </div>
             );
 
+            // instead of showing a url to the gif, show the search phrase as a link to the gif
+            if (GiphyGif.isGiphyLink(this.state.link)) {
```

Comment:
This was my attempt at tidying up the Giphy display. I'm not crazy about it, but I'm also not very good at React.